### DISCUSSION TITLE
How do I submit a feature request to the GitHub team for consideration? <title>

### DISCUSSION THREAD

**Discussion by ptdecker** [community/community#59577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is GitHub open to requests for features and enhancements from the community?  Or, is the organization too big to entertain the voice of individual customers with no time to sort through them, rank them, solicit feedback, etc.?  If there exists a way currently to do it, where do I find that?  The community forums seem to be the place where one learns, "what you ask is how it is designed to behave--I'm closing your comment."

**Comment by jimpriest** [comment]
A year later an no response.  Seems like a closed (and silent) Microsoft universe.  Wish we were using Gitlab :\
